app/javascript/components/search/controls/download/DownloadSelectionModal.js

```diff
@@ -75,14 +122,31 @@ export default function DownloadSelectionModal({ studyAccessions, show, setShow
             to use on your terminal.
           </div>
         </div>
-        { stepNum === 1 && <DownloadSelectionTable
-          isLoading={isLoading}
-          downloadInfo={downloadInfo}
-          selectedBoxes={selectedBoxes}
-          setSelectedBoxes={setSelectedBoxes}/> }
+        { stepNum === 1 &&
+          <div>
+            <DownloadSelectionTable
+              isLoading={isLoading}
+              downloadInfo={downloadInfo}
+              selectedBoxes={selectedBoxes}
+              setSelectedBoxes={setSelectedBoxes}
+              columnTypes={SCP_COLUMNS}/>
+            { showTDRSelectionPane &&
+              <div>
+                <h4>HCA studies</h4>
```

Comment:
Blocker: users often won't intuit the initialism, so let's default to the full name in the UI:

```suggestion
                <h4>Human Cell Atlas studies</h4>
```